src/Compilers/Core/Portable/CommandLine/CommonCompiler.LoggingStrongNameProvider.cs

```diff
@@ -6,12 +6,11 @@ namespace Microsoft.CodeAnalysis
 {
     internal abstract partial class CommonCompiler
     {
-        internal sealed class LoggingStrongNameProvider : DesktopStrongNameProvider
+        internal sealed class LoggingStrongNameFileSystem: StrongNameFileSystem
```

Comment:
space before the colon

Reply:
Ah bummer.  Looks my auto formatter doesn't fix this :(